Add google docs integration (#2635)
automatisch/automatisch: packages/docs/pages/apps/google-docs/triggers.md

---
favicon: /favicons/google-docs.svg
items:

>   - name: New document created
>     desc: Triggers when you create a new Google Doc.

Reviewer: ```suggestion
  - name: New document
    desc: Triggers when a new document is created.
```
